index.html

```diff
@@ -376,7 +376,7 @@
         "#example-4">Example 4</a>, the file picker can be rendered as
         presented below:
       </p>
-      <p>
+      <p style="overflow: visible;">
```

Comment:
I think you would want to use the `overlarge` class (see [Sample use of TR design](https://w3c.github.io/tr-design/src/sample.html#overlarge))

Reply:
The `overlarge` class does not appear to work for this case, since I want the left margin of the image and the surrounding text blocks to be the same, and only overflow the right side of the image that does not fit within the viewport.

Using the `overlarge` class, the left margin of the image differs from that of the surrounding text blocks on various screen widths (depending on which media query kicks in).

I tried applying the class to both the container elements, and separately just to the image elements with similar results.

Reply:
ok, then, LGTM :)